acceptance/config/beaker/local/el6/1host.cfg

```diff
@@ -14,7 +14,7 @@ HOSTS:
     group: puppet
     puppetbin: /usr/bin/puppet
     puppetbindir: /usr/bin
-    puppetvardir: /var/lib/puppet
+    puppetvardir: /opt/puppetlabs/puppet/cache
     hieradatadir: /etc/puppet/hieradata
```

Comment:
Should these hiera and distmoduledir settings be under `/etc/puppetlabs/code` instead of `/etc/puppet`?